### DISCUSSION TITLE
quassel-client does not open a window

### DISCUSSION THREAD

**Discussion by maichai** [Homebrew/discussions#344] [Casks]
On Catalina 10.15.7 the quassel-client does not open a window. I started it also from the shell to see whether it is complaining about anything. It is just missing a few icons (just warnings). Not sure, if that should be a problem.

Ctrl-C does not work, to end quassel-client you need to kill the process. What? Why?

Does anyone get this to work?

PS: I do have a working quassel-core instance on a server and I can use with the quassel web client.

**Comment by SMillerDev** [comment]
What is making you suspect brew as the culprit instead of an upstream bug?

**Comment by maichai** [comment]
if it is an upstream problem, why push an upgrade, if the new version doesn't work?

**Reply by SMillerDev** [reply]
All updates we do run a basic test on CI to make sure our packaging isn't broken. But especially the bug you're describing is hard to test for in an automated way. The responsibility for working software ultimately isn't on homebrew but on the developer of the software.
